### PR TITLE
Add a logging event handler

### PR DESCRIPTION
**Pull request by plettich**
The event handler sends a configurable log message to the python logging
facility whenever the event is triggered (and the conditions match).
The log message can then be processed with a suitable logging
configuration.

Working on #1580

### CODE REVIEW

**Comment by plettich** [comment]
> Please check the PEP8 comments and also add a documentation, which especially should help with:

Done in ba58c86 and e740f12
> 
>     * [x]  Define the logmessage template

I could also add a link to the usernotification handler docs

>     * [ ]  Configure the log config in the YAML file to e.g. pass this specific message to wherever.

I'll add this when this PR is merged because in this branch i don't have the YAML configuration.
This is also the reason why the travis push tests failed, i forgot to add a requirement which is already available in #2080

**Comment by cornelinux** [comment]
@plettich textfixtures is missing and in runs into an import error.

File: tests/test_lib_eventhandler_logging.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+"""
+This file tests:
+
+lib/eventhandler/logging.py
+"""
+from mock import mock
+from datetime import datetime
+from werkzeug.test import EnvironBuilder
+from flask import Request
+from testfixtures import log_capture
```
Comment (cornelinux) [inline tests/test_lib_eventhandler_logging.py]: The testfixtures seems to be missing.

File: doc/eventhandler/logginghandler.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  * ``DEBUG``
+
+**message**
+
+  * *default:* ``"event={action} triggered"``
+
+The message to send to the logging facility. This message can be customized
+with the following tags:
+
+  * ``{admin}``
```
Comment (cornelinux) [inline doc/eventhandler/logginghandler.rst]: Are these always the same tags as in the notification handler?
Then we could in a new PR/issue move the tags to a document section "event handler tags" and simply add a reference here. 
What do you think?

Comment (plettich) [reply doc/eventhandler/logginghandler.rst]: Yes, basically the same but we do not want to include PINs or QR-Codes in the log message.

File: tests/test_lib_eventhandler_logging.py (hunk 1 above)
Comment (plettich) [reply tests/test_lib_eventhandler_logging.py]: Yes, they are already in master from the YAML PR :-)

File: doc/eventhandler/logginghandler.rst (hunk 2 above)
Comment (cornelinux) [reply doc/eventhandler/logginghandler.rst]: Hm, so the build in the master will succeed.

File: tests/test_lib_eventhandler_logging.py (hunk 1 above)
Comment (cornelinux) [reply tests/test_lib_eventhandler_logging.py]: Hm, so the build in the master will succeed.

**Pull request merged**